lib/rubygem_fs.rb

```diff
@@ -70,6 +70,13 @@ def get(key)
       nil
     end
 
+    def deleted?(key)
+      s3.get_object(key: key, bucket: bucket)
```

Comment:
I'd rather use `head_object` as I'm pretty sure this will try to download the file.

http://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Client.html#head_object-instance_method